source/common/ssl/ssl_socket.cc

```diff
@@ -17,6 +17,38 @@ using Envoy::Network::PostIoAction;
 namespace Envoy {
 namespace Ssl {
 
+namespace {
+
+class NotReadySslSocket : public Network::TransportSocket, public Connection {
```

Comment:
Please add comment for this class
// This SslSocket will be used when SSL secret is not fetched from SDS server.

Reply:
Done.